yoti-sdk-spring-boot-example/src/main/resources/templates/fragments.html

```diff
@@ -2,10 +2,12 @@
 <html xmlns:th="http://www.thymeleaf.org">
 <head/>
 <body>
-    <div class="" th:fragment="anchor">
-
-        Hello!
-
+<td th:fragment="anchorDisplay(anchors)">
+    <div th:each="anchor : ${anchors}" style="border: 1px solid grey; display:inline-block; margin-bottom=0" >
```

Comment:
Is it better to move these styles inside a file `style.css` and include it, rather than including them inline?

Reply:
Yes, that would certainly be better, but since this is just an interim measure, let's not got that far.  

We expect to delete this very soon, as soon as we get the new demo page.